mettle/main.go

```diff
@@ -76,8 +78,9 @@ var opts = struct {
 	} `command:"worker" description:"Start as a worker"`
 	Dual struct {
 		GRPC            grpcutil.Opts           `group:"Options controlling the gRPC server"`
-		Dir             string                  `short:"d" long:"dir" default:"plz-out/mettle" description:"Directory to run actions in"`
+		Dir             string                  `short:"d" long:"dir" default:"plz-out/mettle" description:"Base directory to run actions in"`
```

Comment:
Minor one I found while testing this: all the workers share the same temp directory in this setup. They don't normally clash because working dirs are named after the action digest, and we deduplicate actions, but it happens with the preflight check because it's the same one on each worker.

I don't think this would be an issue often otherwise, but I'm giving each its own directory for good measure.